src/sonic-bgpcfgd/tests/test_bgp.py

```diff
@@ -38,75 +46,89 @@ def constructor():
+    for constant in load_constant_files():
+        m = constructor(constant)
+        res = m.set_handler("fc00:10::1", {"admin_status": "up"})
+        assert res, "Expect True return value for peer update"
+        mocked_log_info.assert_called_with("Peer 'default|fc00:10::1' admin state is set to 'up'")
 
 @patch('bgpcfgd.managers_bgp.log_info')
 def test_update_peer_down(mocked_log_info): 
-    m = constructor()
-    res = m.set_handler("10.10.10.1", {"admin_status": "down"})
-    assert res, "Expect True return value for peer update"
-    mocked_log_info.assert_called_with("Peer 'default|10.10.10.1' admin state is set to 'down'")
+    for constant in load_constant_files():
+        m = constructor(constant)
+        res = m.set_handler("10.10.10.1", {"admin_status": "down"})
+        assert res, "Expect True return value for peer update"
+        mocked_log_info.assert_called_with("Peer 'default|10.10.10.1' admin state is set to 'down'")
 
 @patch('bgpcfgd.managers_bgp.log_err')
 def test_update_peer_no_admin_status(mocked_log_err):
-    m = constructor()
-    res = m.set_handler("10.10.10.1", {"anything": "anything"})
-    assert res, "Expect True return value for peer update"
-    mocked_log_err.assert_called_with("Peer '(default|10.10.10.1)': Can't update the peer. Only 'admin_status' attribute is supported")
+    for constant in load_constant_files():
+        m = constructor(constant)
+        res = m.set_handler("10.10.10.1", {"anything": "anything"})
+        assert res, "Expect True return value for peer update"
+        mocked_log_err.assert_called_with("Peer '(default|10.10.10.1)': Can't update the peer. Only 'admin_status' attribute is supported")
 
 @patch('bgpcfgd.managers_bgp.log_err')
 def test_update_peer_invalid_admin_status(mocked_log_err):
-    m = constructor()
-    res = m.set_handler("10.10.10.1", {"admin_status": "invalid"})
-    assert res, "Expect True return value for peer update"
-    mocked_log_err.assert_called_with("Peer 'default|10.10.10.1': Can't update the peer. It has wrong attribute value attr['admin_status'] = 'invalid'")
+    for constant in load_constant_files():
+        m = constructor(constant)
+        res = m.set_handler("10.10.10.1", {"admin_status": "invalid"})
+        assert res, "Expect True return value for peer update"
+        mocked_log_err.assert_called_with("Peer 'default|10.10.10.1': Can't update the peer. It has wrong attribute value attr['admin_status'] = 'invalid'")
 
 def test_add_peer():
-    m = constructor()
-    res = m.set_handler("30.30.30.1", {"local_addr": "30.30.30.30", "admin_status": "up"})
-    assert res, "Expect True return value"
+    for constant in load_constant_files():
+        m = constructor(constant)
+        res = m.set_handler("30.30.30.1", {'asn': '65200', 'holdtime': '180', 'keepalive': '60', 'local_addr': '30.30.30.30', 'name': 'TOR', 'nhopself': '0', 'rrclient': '0'})
```

Comment:
@geogchen this one "admin_status": "up" is missing, what's the reason we don't have this in the add peer test?

Reply:
when I did trace over the add_peer call, I did not see admin status included.